### PR TITLE
Implement Statement support on DremioFlightProducer

### PR DESCRIPTION
**Pull request by rafael-telles**
This implements `CommandStatementQuery` on DremioFlightProducer.

Other changes in this PR:
- Update with last version of `FlightSqlClient`
- Move unit tests of catalog functions and refactor top-level test classes to test both Statement and PreparedStatement query executions

### CODE REVIEW

File: services/arrow-flight/src/main/java/com/dremio/service/flight/DremioFlightProducer.java [hunk 1]
```diff
@@ -221,12 +225,11 @@ public void createPreparedStatement(
     ActionCreatePreparedStatementRequest actionCreatePreparedStatementRequest,
     CallContext callContext,
     StreamListener<Result> streamListener) {
-    final FlightDescriptor flightDescriptor =
-      FlightDescriptor.command(actionCreatePreparedStatementRequest.getQuery().getBytes(StandardCharsets.UTF_8));
+    String query = actionCreatePreparedStatementRequest.getQuery();
```
Comment (normanj-bitquill) [inline services/arrow-flight/src/main/java/com/dremio/service/flight/DremioFlightProducer.java:228]: final

File: services/arrow-flight/src/main/java/com/dremio/service/flight/DremioFlightProducer.java [hunk 2]
```diff
@@ -256,21 +259,42 @@ public void closePreparedStatement(
...
 
   @Override
-  public void getStreamStatement(CommandStatementQuery commandStatementQuery,
-                                 CallContext callContext, Ticket ticket,
+  public void getStreamStatement(TicketStatementQuery commandStatementQuery,
+                                 CallContext callContext,
                                  ServerStreamListener serverStreamListener) {
-    throw CallStatus.UNIMPLEMENTED.withDescription("Statement not supported.").toRuntimeException();
+    try {
+      UserProtos.PreparedStatementHandle preparedStatementHandle =
```
Comment (normanj-bitquill) [inline services/arrow-flight/src/main/java/com/dremio/service/flight/DremioFlightProducer.java:291]: final

File: services/arrow-flight/src/main/java/com/dremio/service/flight/DremioFlightProducer.java [hunk 3]
```diff
@@ -256,21 +259,42 @@ public void closePreparedStatement(
...
+    Schema schema = flightPreparedStatement.getSchema();
+    return getFlightInfoForFlightSqlCommands(ticket, flightDescriptor, schema);
   }
 
   @Override
   public SchemaResult getSchemaStatement(
     CommandStatementQuery commandStatementQuery,
     CallContext callContext, FlightDescriptor flightDescriptor) {
-    throw CallStatus.UNIMPLEMENTED.withDescription("Statement not supported.").toRuntimeException();
+    FlightInfo info = this.getFlightInfo(callContext, flightDescriptor);
```
Comment (normanj-bitquill) [inline services/arrow-flight/src/main/java/com/dremio/service/flight/DremioFlightProducer.java:282]: final

File: services/arrow-flight/src/main/java/com/dremio/service/flight/DremioFlightProducer.java [hunk 4]
```diff
@@ -256,21 +259,42 @@ public void closePreparedStatement(
...
+      .createPreparedStatement(commandStatementQuery.getQuery(), callContext::isCancelled, session);
+
+    flightPreparedStatementCache.put(flightPreparedStatement.getServerHandle(), flightPreparedStatement);
+
+    TicketStatementQuery ticket =
+      TicketStatementQuery.newBuilder()
+        .setStatementHandle(flightPreparedStatement.getServerHandle().toByteString())
+        .build();
+
+    Schema schema = flightPreparedStatement.getSchema();
```
Comment (normanj-bitquill) [inline services/arrow-flight/src/main/java/com/dremio/service/flight/DremioFlightProducer.java:274]: final

File: services/arrow-flight/src/main/java/com/dremio/service/flight/DremioFlightProducer.java [hunk 5]
```diff
@@ -256,21 +259,42 @@ public void closePreparedStatement(
...
     CallContext callContext, FlightDescriptor flightDescriptor) {
-    throw CallStatus.UNIMPLEMENTED.withDescription("Statement not supported.").toRuntimeException();
+    final UserSession session = getUserSessionFromCallContext(callContext);
+
+    final FlightPreparedStatement flightPreparedStatement = flightWorkManager
+      .createPreparedStatement(commandStatementQuery.getQuery(), callContext::isCancelled, session);
+
+    flightPreparedStatementCache.put(flightPreparedStatement.getServerHandle(), flightPreparedStatement);
+
+    TicketStatementQuery ticket =
```
Comment (normanj-bitquill) [inline services/arrow-flight/src/main/java/com/dremio/service/flight/DremioFlightProducer.java:269]: final

File: services/arrow-flight/src/main/java/com/dremio/service/flight/impl/FlightPreparedStatement.java [hunk 6]
```diff
@@ -57,12 +57,13 @@ public FlightInfo getFlightInfo(Location location) {
     final UserProtos.CreatePreparedStatementArrowResp createPreparedStatementResp = responseHandler.get();
     final Schema schema = buildSchema(createPreparedStatementResp.getPreparedStatement().getArrowSchema());
 
-    final PreparedStatementTicket preparedStatementTicketContent = PreparedStatementTicket.newBuilder()
-      .setQuery(query)
-      .setHandle(getServerHandle())
-      .build();
+    UserProtos.PreparedStatementHandle preparedStatementHandle = getServerHandle();
```
Comment (normanj-bitquill) [inline services/arrow-flight/src/main/java/com/dremio/service/flight/impl/FlightPreparedStatement.java:60]: final

File: services/arrow-flight/src/test/java/com/dremio/service/flight/AbstractTestFlightSqlServer.java [hunk 7]
```diff
@@ -38,194 +44,43 @@
...
-      null, false, getCallOptions());
-    try (FlightStream stream = flightSqlClient.getStream(flightInfo.getEndpoints().get(0).getTicket(),
-      getCallOptions())) {
-      Assert.assertTrue(stream.next());
-      VectorSchemaRoot root = stream.getRoot();
-
-      Assert.assertEquals(root.getRowCount(), 28);
-    }
+  private FlightInfo executeStatement(String query) {
+    FlightSqlClient client = getFlightClientWrapper().getSqlClient();
```
Comment (normanj-bitquill) [inline services/arrow-flight/src/test/java/com/dremio/service/flight/AbstractTestFlightSqlServer.java:67]: final

File: services/arrow-flight/src/test/java/com/dremio/service/flight/AbstractTestFlightSqlServer.java [hunk 8]
```diff
@@ -38,194 +44,43 @@
...
-      final IntStream range = IntStream.range(0, counter);
-
-      Assert.assertEquals(root.getRowCount(), counter);
-      range.forEach(i -> {
-        Assert.assertEquals(root.getVector(0).getObject(i).toString(), tableTypes.get(i).toString());
-      });
-    }
+  private FlightInfo executePreparedStatement(String query) throws SQLException {
+    FlightSqlClient client = getFlightClientWrapper().getSqlClient();
+    FlightSqlClient.PreparedStatement preparedStatement = client.prepare(query, getCallOptions());
```
Comment (normanj-bitquill) [inline services/arrow-flight/src/test/java/com/dremio/service/flight/AbstractTestFlightSqlServer.java:73]: final

File: services/arrow-flight/src/test/java/com/dremio/service/flight/AbstractTestFlightSqlServerCatalogMethods.java [hunk 9]
```diff
@@ -0,0 +1,224 @@
...
+import com.dremio.service.catalog.TableType;
+import com.google.common.collect.ImmutableList;
+
+public abstract class AbstractTestFlightSqlServerCatalogMethods extends BaseFlightQueryTest {
+
+  abstract CallOption[] getCallOptions();
+
+  @Test
+  public void testGetCatalogs() throws Exception {
+    FlightSqlClient flightSqlClient = getFlightClientWrapper().getSqlClient();
```
Comment (normanj-bitquill) [inline services/arrow-flight/src/test/java/com/dremio/service/flight/AbstractTestFlightSqlServerCatalogMethods.java:45]: final

File: services/arrow-flight/src/test/java/com/dremio/service/flight/AbstractTestFlightSqlServerCatalogMethods.java [hunk 10]
```diff
@@ -0,0 +1,224 @@
...
+    CallOption[] callOptions = getCallOptions();
+
+    FlightInfo flightInfo = flightSqlClient.getCatalogs(callOptions);
```
Comment (normanj-bitquill) [inline services/arrow-flight/src/test/java/com/dremio/service/flight/AbstractTestFlightSqlServerCatalogMethods.java:48]: final

File: services/arrow-flight/src/test/java/com/dremio/service/flight/AbstractTestFlightSqlServerCatalogMethods.java [hunk 11]
```diff
@@ -0,0 +1,224 @@
...
+    try (FlightStream stream = flightSqlClient.getStream(flightInfo.getEndpoints().get(0).getTicket(), callOptions)) {
+      Assert.assertTrue(stream.next());
+      VectorSchemaRoot root = stream.getRoot();
```
Comment (normanj-bitquill) [inline services/arrow-flight/src/test/java/com/dremio/service/flight/AbstractTestFlightSqlServerCatalogMethods.java:51]: final

File: services/arrow-flight/src/test/java/com/dremio/service/flight/AbstractTestFlightSqlServerCatalogMethods.java [hunk 12]
```diff
@@ -0,0 +1,224 @@
...
+      Assert.assertEquals(1, root.getRowCount());
+
+      String catalogName = ((VarCharVector) root.getVector("catalog_name")).getObject(0).toString();
```
Comment (normanj-bitquill) [inline services/arrow-flight/src/test/java/com/dremio/service/flight/AbstractTestFlightSqlServerCatalogMethods.java:54]: final

File: services/arrow-flight/src/test/java/com/dremio/service/flight/AbstractTestFlightSqlServerCatalogMethods.java [hunk 13]
```diff
@@ -0,0 +1,224 @@
...
+      Assert.assertEquals("DREMIO", catalogName);
+    }
+  }
+
+  @Test
+  public void testGetTablesWithoutFiltering() throws Exception {
+    FlightSqlClient flightSqlClient = getFlightClientWrapper().getSqlClient();
+    FlightInfo flightInfo = flightSqlClient.getTables(null, null, null,
```
Comment (normanj-bitquill) [inline services/arrow-flight/src/test/java/com/dremio/service/flight/AbstractTestFlightSqlServerCatalogMethods.java:62]: final

File: services/arrow-flight/src/test/java/com/dremio/service/flight/AbstractTestFlightSqlServerCatalogMethods.java [hunk 14]
```diff
@@ -0,0 +1,224 @@
...
+      null, false, getCallOptions());
+    try (
+      FlightStream stream = flightSqlClient.getStream(flightInfo.getEndpoints().get(0).getTicket(), getCallOptions())) {
```
Comment (normanj-bitquill) [inline services/arrow-flight/src/test/java/com/dremio/service/flight/AbstractTestFlightSqlServerCatalogMethods.java:65]: final

File: services/arrow-flight/src/test/java/com/dremio/service/flight/AbstractTestFlightSqlServerCatalogMethods.java [hunk 15]
```diff
@@ -0,0 +1,224 @@
...
+      Assert.assertTrue(stream.next());
+      VectorSchemaRoot root = stream.getRoot();
```
Comment (normanj-bitquill) [inline services/arrow-flight/src/test/java/com/dremio/service/flight/AbstractTestFlightSqlServerCatalogMethods.java:67]: final

File: services/arrow-flight/src/test/java/com/dremio/service/flight/AbstractTestFlightSqlServerCatalogMethods.java [hunk 16]
```diff
@@ -0,0 +1,224 @@
...
+
+      Assert.assertEquals(root.getRowCount(), 28);
+    }
+  }
+
+  @Test
+  public void testGetTablesFilteringByCatalogPattern() throws Exception {
+    FlightSqlClient flightSqlClient = getFlightClientWrapper().getSqlClient();
+    FlightInfo flightInfo = flightSqlClient.getTables("DREMIO", null, null,
```
Comment (normanj-bitquill) [inline services/arrow-flight/src/test/java/com/dremio/service/flight/AbstractTestFlightSqlServerCatalogMethods.java:76]: final

File: services/arrow-flight/src/main/java/com/dremio/service/flight/DremioFlightProducer.java [hunk 17]
```diff
@@ -256,21 +259,42 @@ public void closePreparedStatement(
...
-    throw CallStatus.UNIMPLEMENTED.withDescription("Statement not supported.").toRuntimeException();
+    FlightInfo info = this.getFlightInfo(callContext, flightDescriptor);
+    return new SchemaResult(info.getSchema());
   }
 
   @Override
-  public void getStreamStatement(CommandStatementQuery commandStatementQuery,
-                                 CallContext callContext, Ticket ticket,
+  public void getStreamStatement(TicketStatementQuery commandStatementQuery,
+                                 CallContext callContext,
```
Comment (kylep-dremio) [inline services/arrow-flight/src/main/java/com/dremio/service/flight/DremioFlightProducer.java:288]: What happened to the `Ticket ticket` parameter?

File: services/arrow-flight/src/test/java/com/dremio/service/flight/AbstractTestFlightSqlServerCatalogMethods.java [hunk 18]
```diff
@@ -0,0 +1,224 @@
...
+      null, false, getCallOptions());
+    try (FlightStream stream = flightSqlClient.getStream(flightInfo.getEndpoints().get(0).getTicket(),
+      getCallOptions())) {
+      Assert.assertTrue(stream.next());
+      VectorSchemaRoot root = stream.getRoot();
```
Comment (normanj-bitquill) [inline services/arrow-flight/src/test/java/com/dremio/service/flight/AbstractTestFlightSqlServerCatalogMethods.java:81]: final

File: services/arrow-flight/src/test/java/com/dremio/service/flight/AbstractTestFlightSqlServerCatalogMethods.java [hunk 19]
```diff
@@ -0,0 +1,224 @@
...
+
+      Assert.assertEquals(root.getRowCount(), 28);
+    }
+  }
+
+  @Test
+  public void testGetTablesFilteringBySchemaPattern() throws Exception {
+    FlightSqlClient flightSqlClient = getFlightClientWrapper().getSqlClient();
+    FlightInfo flightInfo = flightSqlClient.getTables(null, "INFORMATION_SCHEMA",
```
Comment (normanj-bitquill) [inline services/arrow-flight/src/test/java/com/dremio/service/flight/AbstractTestFlightSqlServerCatalogMethods.java:90]: final

File: services/arrow-flight/src/test/java/com/dremio/service/flight/AbstractTestFlightSqlServerCatalogMethods.java [hunk 20]
```diff
@@ -0,0 +1,224 @@
...
+      null, null, false, getCallOptions());
+    try (FlightStream stream = flightSqlClient.getStream(flightInfo.getEndpoints().get(0).getTicket(),
+      getCallOptions())) {
+      Assert.assertTrue(stream.next());
+      VectorSchemaRoot root = stream.getRoot();
```
Comment (normanj-bitquill) [inline services/arrow-flight/src/test/java/com/dremio/service/flight/AbstractTestFlightSqlServerCatalogMethods.java:95]: final

File: services/arrow-flight/src/test/java/com/dremio/service/flight/AbstractTestFlightSqlServerCatalogMethods.java [hunk 21]
```diff
@@ -0,0 +1,224 @@
...
+
+      Assert.assertEquals(root.getRowCount(), 5);
+    }
+  }
+
+  @Test
+  public void testGetTablesFilteringByTablePattern() throws Exception {
+    FlightSqlClient flightSqlClient = getFlightClientWrapper().getSqlClient();
+    FlightInfo flightInfo = flightSqlClient.getTables(null, null, "COLUMNS",
```
Comment (normanj-bitquill) [inline services/arrow-flight/src/test/java/com/dremio/service/flight/AbstractTestFlightSqlServerCatalogMethods.java:104]: final

File: services/arrow-flight/src/test/java/com/dremio/service/flight/AbstractTestFlightSqlServerCatalogMethods.java [hunk 22]
```diff
@@ -0,0 +1,224 @@
...
+      null, false, getCallOptions());
+    try (FlightStream stream = flightSqlClient.getStream(flightInfo.getEndpoints().get(0).getTicket(),
+      getCallOptions())) {
+      Assert.assertTrue(stream.next());
+      VectorSchemaRoot root = stream.getRoot();
```
Comment (normanj-bitquill) [inline services/arrow-flight/src/test/java/com/dremio/service/flight/AbstractTestFlightSqlServerCatalogMethods.java:109]: final

File: services/arrow-flight/src/test/java/com/dremio/service/flight/AbstractTestFlightSqlServerCatalogMethods.java [hunk 23]
```diff
@@ -0,0 +1,224 @@
...
+
+      Assert.assertEquals(root.getRowCount(), 1);
+    }
+  }
+
+  @Test
+  public void testGetTablesFilteringByTableTypePattern() throws Exception {
+    FlightSqlClient flightSqlClient = getFlightClientWrapper().getSqlClient();
+    FlightInfo flightInfo = flightSqlClient.getTables(null, null, null,
```
Comment (normanj-bitquill) [inline services/arrow-flight/src/test/java/com/dremio/service/flight/AbstractTestFlightSqlServerCatalogMethods.java:118]: final

File: services/arrow-flight/src/test/java/com/dremio/service/flight/AbstractTestFlightSqlServerCatalogMethods.java [hunk 24]
```diff
@@ -0,0 +1,224 @@
...
+      Collections.singletonList("SYSTEM_TABLE"), false, getCallOptions());
+    try (FlightStream stream = flightSqlClient.getStream(flightInfo.getEndpoints().get(0).getTicket(),
+      getCallOptions())) {
+      Assert.assertTrue(stream.next());
+      VectorSchemaRoot root = stream.getRoot();
```
Comment (normanj-bitquill) [inline services/arrow-flight/src/test/java/com/dremio/service/flight/AbstractTestFlightSqlServerCatalogMethods.java:123]: final

File: services/arrow-flight/src/test/java/com/dremio/service/flight/AbstractTestFlightSqlServerCatalogMethods.java [hunk 25]
```diff
@@ -0,0 +1,224 @@
...
+
+      Assert.assertEquals(root.getRowCount(), 28);
+    }
+  }
+
+  @Test
+  public void testGetTablesFilteringByMultiTableTypes() throws Exception {
+    FlightSqlClient flightSqlClient = getFlightClientWrapper().getSqlClient();
+    FlightInfo flightInfo = flightSqlClient.getTables(null, null, null,
```
Comment (normanj-bitquill) [inline services/arrow-flight/src/test/java/com/dremio/service/flight/AbstractTestFlightSqlServerCatalogMethods.java:132]: final

File: services/arrow-flight/src/test/java/com/dremio/service/flight/AbstractTestFlightSqlServerCatalogMethods.java [hunk 26]
```diff
@@ -0,0 +1,224 @@
...
+      ImmutableList.of("TABLE", "VIEW"), false, getCallOptions());
+    try (FlightStream stream = flightSqlClient.getStream(flightInfo.getEndpoints().get(0).getTicket(),
+      getCallOptions())) {
+      Assert.assertTrue(stream.next());
+      VectorSchemaRoot root = stream.getRoot();
```
Comment (normanj-bitquill) [inline services/arrow-flight/src/test/java/com/dremio/service/flight/AbstractTestFlightSqlServerCatalogMethods.java:137]: final

File: services/arrow-flight/src/test/java/com/dremio/service/flight/AbstractTestFlightSqlServerCatalogMethods.java [hunk 27]
```diff
@@ -0,0 +1,224 @@
...
+
+      Assert.assertEquals(root.getRowCount(), 0);
+    }
+  }
+
+  @Test
+  public void testGetTablesTypes() throws Exception {
+    FlightSqlClient flightSqlClient = getFlightClientWrapper().getSqlClient();
+    FlightInfo flightInfo = flightSqlClient.getTableTypes(getCallOptions());
```
Comment (normanj-bitquill) [inline services/arrow-flight/src/test/java/com/dremio/service/flight/AbstractTestFlightSqlServerCatalogMethods.java:146]: final

File: services/arrow-flight/src/test/java/com/dremio/service/flight/AbstractTestFlightSqlServerCatalogMethods.java [hunk 28]
```diff
@@ -0,0 +1,224 @@
...
+    try (
+      FlightStream stream = flightSqlClient.getStream(flightInfo.getEndpoints().get(0).getTicket(), getCallOptions())) {
```
Comment (normanj-bitquill) [inline services/arrow-flight/src/test/java/com/dremio/service/flight/AbstractTestFlightSqlServerCatalogMethods.java:148]: final

File: services/arrow-flight/src/test/java/com/dremio/service/flight/AbstractTestFlightSqlServerCatalogMethods.java [hunk 29]
```diff
@@ -0,0 +1,224 @@
...
+      Assert.assertTrue(stream.next());
+      VectorSchemaRoot root = stream.getRoot();
```
Comment (normanj-bitquill) [inline services/arrow-flight/src/test/java/com/dremio/service/flight/AbstractTestFlightSqlServerCatalogMethods.java:150]: final

File: services/arrow-flight/src/test/java/com/dremio/service/flight/AbstractTestFlightSqlServerCatalogMethods.java [hunk 30]
```diff
@@ -0,0 +1,224 @@
...
+      Assert.assertEquals(root.getRowCount(), counter);
+      range.forEach(i -> {
+        Assert.assertEquals(root.getVector(0).getObject(i).toString(), tableTypes.get(i).toString());
+      });
+    }
+  }
+
+  private void testGetSchemas(String catalog, String schemaPattern, boolean expectNonEmptyResult) throws Exception {
+    FlightSqlClient flightSqlClient = getFlightClientWrapper().getSqlClient();
+    FlightInfo flightInfo = flightSqlClient.getSchemas(catalog, schemaPattern, getCallOptions());
```
Comment (normanj-bitquill) [inline services/arrow-flight/src/test/java/com/dremio/service/flight/AbstractTestFlightSqlServerCatalogMethods.java:167]: final.

Also this is coming up a lot. Should it go in the @Before method and promote these to member variables?

File: services/arrow-flight/src/test/java/com/dremio/service/flight/AbstractTestFlightSqlServerCatalogMethods.java [hunk 31]
```diff
@@ -0,0 +1,224 @@
...
+    try (
+      FlightStream stream = flightSqlClient.getStream(flightInfo.getEndpoints().get(0).getTicket(), getCallOptions())) {
```
Comment (normanj-bitquill) [inline services/arrow-flight/src/test/java/com/dremio/service/flight/AbstractTestFlightSqlServerCatalogMethods.java:169]: final

File: services/arrow-flight/src/test/java/com/dremio/service/flight/AbstractTestFlightSqlServerCatalogMethods.java [hunk 32]
```diff
@@ -0,0 +1,224 @@
...
+      Assert.assertTrue(stream.next());
+      VectorSchemaRoot root = stream.getRoot();
```
Comment (normanj-bitquill) [inline services/arrow-flight/src/test/java/com/dremio/service/flight/AbstractTestFlightSqlServerCatalogMethods.java:171]: final

File: services/arrow-flight/src/test/java/com/dremio/service/flight/AbstractTestFlightSqlServerCatalogMethods.java [hunk 33]
```diff
@@ -0,0 +1,224 @@
...
+
+      Predicate<String> catalogNamePredicate = MetadataProviderConditions.getCatalogNamePredicate(
+        catalog != null ? UserProtos.LikeFilter.newBuilder().setPattern(catalog).build() : null);
+      Pattern schemaFilterPattern = schemaPattern != null ? Pattern.compile(RegexpUtil.sqlToRegexLike(schemaPattern)) :
+        Pattern.compile(".*");
+
+      VarCharVector catalogNameVector = (VarCharVector) root.getVector("catalog_name");
+      VarCharVector schemaNameVector = (VarCharVector) root.getVector("schema_name");
```
Comment (normanj-bitquill) [inline services/arrow-flight/src/test/java/com/dremio/service/flight/AbstractTestFlightSqlServerCatalogMethods.java:179]: final

File: services/arrow-flight/src/test/java/com/dremio/service/flight/AbstractTestFlightSqlServerCatalogMethods.java [hunk 34]
```diff
@@ -0,0 +1,224 @@
...
+
+      for (int i = 0; i < root.getRowCount(); i++) {
+        String catalogName = catalogNameVector.getObject(i).toString();
+        String schemaName = schemaNameVector.getObject(i).toString();
```
Comment (normanj-bitquill) [inline services/arrow-flight/src/test/java/com/dremio/service/flight/AbstractTestFlightSqlServerCatalogMethods.java:183]: final

File: services/arrow-flight/src/main/java/com/dremio/service/flight/DremioFlightProducer.java (hunk 17 above)
Comment (rafael-telles) [reply services/arrow-flight/src/main/java/com/dremio/service/flight/DremioFlightProducer.java:287]: Fixed - that was missing on the current flight-sql version I had built

File: services/arrow-flight/src/test/java/com/dremio/service/flight/AbstractTestFlightSqlServerCatalogMethods.java (hunk 30 above)
Comment (rafael-telles) [reply services/arrow-flight/src/test/java/com/dremio/service/flight/AbstractTestFlightSqlServerCatalogMethods.java]: Fixed

**Review by normanj-bitquill — APPROVED** [review]

**Pull request merged**